### PR TITLE
FTP Provider: Send filename instead of full path

### PR DESCRIPTION
**Pull request by zuntah**

### CODE REVIEW

File: packages/nexrender-provider-ftp/src/index.js [hunk 1]
```diff
@@ -47,7 +47,8 @@ const upload = (job, settings, src, params) => {
         file.on('error', (err) => reject(err))
 
         con.connect(params);
-        con.put(file, src, function(err) {
+        var filename = src.replace(/^.*[\\\/]/, '')
```
Comment (TimP4w) [inline packages/nexrender-provider-ftp/src/index.js]: This is not the filename, but the whole path stripped of slashes. https://nodejs.org/api/path.html#path_path_basename_path_ext may be a better and safer option.

Comment (inlife) [reply packages/nexrender-provider-ftp/src/index.js]: @zuntah I agree with Tim, using regex for such stuff might be an additional cause for future bugs

Methods from `path` would be indeed much more safer

**Comment by zuntah** [comment]
Done :)

**Comment by inlife** [comment]
Great, thank you!

**Pull request merged**